tests/fastnumberpathtests.cpp

```diff
@@ -0,0 +1,134 @@
+#include <cassert>
+#include <cinttypes>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "simdjson/jsonparser.h"
+
+static inline uint64_t splitmix64_stateless(uint64_t index) {
```

Comment:
It'd be worthwhile to comment this method the same way this blog post was--it took me some googling to understand what this was for, since the method doesn't really say anything about it being a random number seed and the method certainly doesn't self-explain :)